Resources/views/webModules/MTExtranet/accessDenied.view.php

```diff
@@ -16,11 +17,23 @@
 $oViewRenderer->AddSourceObject('sForgetPasswordLinkTitle', $translator->trans('chameleon_system_chameleon_shop_theme.extranet.question_reset_password'));
 
 if (empty($sSuccessURL)) {
-    $activePageService = \ChameleonSystem\CoreBundle\ServiceLocator::get('chameleon_system_core.active_page_service');
+    /**
+     * @var $activePageService ActivePageServiceInterface
+     */
+    $activePageService = ServiceLocator::get('chameleon_system_core.active_page_service');
     $sSuccessURL = $activePageService->getLinkToActivePageRelative();
 }
 $oViewRenderer->AddSourceObject('sLoginSuccessURL', $sSuccessURL);
 
+/**
+ * @var $extranetConfig ExtranetConfigurationInterface
+ */
+$extranetConfig = ServiceLocator::get('chameleon_system_extranet.extranet_config');
+$loginUrl = $extranetConfig->getLink(ExtranetConfigurationInterface::PAGE_LOGIN);
+if (null !== $loginUrl && '' !== $loginUrl) {
+    $oViewRenderer->AddSourceObject('loginFormAction', $loginUrl);
```

Comment:
What is the intention behind explicitly setting the form action?

Reply:
Why in the success url?

Above that is the current (intially requested) url.

Reply:
Nevermind, looks good.